### ISSUE TITLE
Unable to install extension/theme

### ISSUE THREAD

**Issue by saegl5** [standardnotes/desktop#550]
Hi there. After installing Standard Notes 3.4.1, I encountered an identical error mentioned in issue #521. Same error message but for installing extensions/themes.

In the meantime, I have uninstalled Standard Notes 3.4.1 and reinstalled version 3.3.5.
Version 3.3.5 works fine.

**Comment by JaspalSuri** [comment]
That's strange. Do you recall what extensions you were trying to install? What OS are you using?

**Comment by saegl5** [comment]
Hi @JaspalSuri,
I was trying to install a custom extension, my own extension/theme actually, using windows 10.

Found the issue: The download_url was referencing a zipped archive in GitLab.
I know!~~~~ [Publishing directions](https://docs.standardnotes.org/extensions/publishing) do recommend using GitHub, but I prefer GitLab and it did work until version 3.4.1.

So, I reinstalled version 3.4.1 but mirrored my repository from GitLab to GitHub and changed the download_url accordingly.
Version 3.4.1 works, now.
